core/server/master/src/test/java/alluxio/master/metastore/caching/CachingInodeStoreMockedBackingStoreTest.java

```diff
@@ -335,6 +336,27 @@ public void skipCache() throws Exception {
     assertEquals(0, mStore.mInodeCache.getCacheMap().size());
   }
 
+  @Test
+  public void evictInodeForTestHasChildren() throws Exception {
```

Comment:
thanks for adding this test case! My understanding is, once your code is merged, this test case is no longer needed right? In other words, if your code is merged, `assertEquals(false, mStore.hasChildren(myTestdir));` will return false.